### PR TITLE
Dashboard alterations

### LINKED ISSUE #93: Dashboard - add Bandcamp URLs to bar charts
Add URLs to Altair charts so that people can click on a bar to be taken to the relavent page on bandcamp.

As an engineer I need to make the UI of the dashboard the most useful considering the target audience and what they would like.

### PR DESCRIPTION
**Pull request by alina-101**
Edited queries to pass URLs
Added URLs to bar charts so you can click on the bars to be taken to relevant pages.
Also altered colour schemes for bar charts

closes #93

### CODE REVIEW

File: dashboard/pages/Artists.py [hunk 1]
```diff
@@ -11,6 +11,8 @@ def show_artists():
     conn = get_connection()
     pop_artists = get_popular_artists(conn)
     st.header("Top Artists")
+    st.write("Click on the bar to be taken to the relavant page on Bandcamp")
```
Comment (e-lemma) [inline dashboard/pages/Artists.py:14]:
```suggestion
    st.write("Click on the bar to be taken to the relavant page on Bandcamp")
```
```suggestion
    st.write("Click on the bar to be taken to the relevant page on Bandcamp")
```

File: dashboard/pages/Home.py [hunk 2]
```diff
@@ -19,11 +19,12 @@ def show_home():
     tracks = database.get_popular_tracks(conn)
     albums = database.get_popular_albums(conn)
     st.subheader("Top Tracks")
+    st.write("Click on the bar to be taken to the relavant page on Bandcamp")
```
Comment (e-lemma) [inline dashboard/pages/Home.py:22]:
```suggestion
    st.write("Click on the bar to be taken to the relavant page on Bandcamp")
```
```suggestion
    st.write("Click on the bar to be taken to the relevant page on Bandcamp")
```

File: dashboard/pages/Home.py [hunk 3]
```diff
@@ -19,11 +19,12 @@ def show_home():
...
+
     st.altair_chart(charts.get_most_copies_sold_chart(
         tracks), use_container_width=True)
     st.subheader("Top Albums")
+    st.write("Click on the bar to be taken to the relavant page on Bandcamp")
```
Comment (e-lemma) [inline dashboard/pages/Home.py:27]:
```suggestion
    st.write("Click on the bar to be taken to the relevant page on Bandcamp")
```

File: dashboard/charts.py [hunk 4]
```diff
@@ -11,28 +11,32 @@ def get_most_copies_sold_chart(data: DataFrame) -> alt.Chart:
...
     )
 
 
 @st.cache_data
 def get_most_popular_tags_chart(tags: DataFrame) -> alt.Chart:
     """Returns a bar chart of popular tags and their sales."""
 
-    return alt.Chart(tags).mark_bar().encode(
-        x=alt.X("name:N", title="Tags"),
+    return alt.Chart(tags).mark_bar(color="rgb(60, 154, 170)").encode(
```
Comment (e-lemma) [inline dashboard/charts.py:38]: Why have you chosen this colour in particular?

Comment (alina-101) [reply dashboard/charts.py:38]: This is the Bandcamp logo colour. It wasn't available as one of the given colours within Altair so I added it by its RGB colour code

**Pull request merged**